res/skins/QMLDemo/Deck.qml

```diff
@@ -13,6 +13,20 @@ Item {
     property bool minimized: false
     property var deckPlayer: Mixxx.PlayerManager.getPlayer(group)
 
+    Drag.active: dragArea.drag.active
```

Comment:
This is causing problems:
> warning [Main] file:///home/swiftb0y/Sourcerepositories/mixxx/res/skins/QMLDemo/DeckRow.qml:71:5: QML Deck (parent or ancestor of QQuickDragAttached): Binding loop detected for property "active"

You can reproduce it by dragging the deck to any Item that has no PlayerDropArea, and then clicking anywhere else in the skin.

Reply:
I cannot reproduce this. It's not even possible to drag stuff anywhere else, the mouse pointer because a "no waiting" road sign.

Reply:
Btw, that line is taken from the QML Drag & Drop example: https://doc.qt.io/qt-5/qtquick-draganddrop-example.html

Reply:
> I cannot reproduce this. It's not even possible to drag stuff anywhere else, the mouse pointer becomes a "no waiting" road sign.


Mhmm. I don't even have a change in mouse pointer. I guess its an underlying bug with Qt and the window manager or something?

Reply:
🤷‍♂️

Reply:
We can worry about this some other time.